apps/aurora-portal/src/client/Identity/Auth/Nav/Nav.tsx

```diff
@@ -0,0 +1,38 @@
+import { useLocation } from "wouter"
+// @ts-expect-error missing types
+import { Button, Icon } from "@cloudoperators/juno-ui-components"
+import { trpc } from "../../../trpcClient"
+
+export function Nav() {
+  const checkLoginStatus = trpc.identity.getAuthStatus.useQuery()
+  const logoutMutation = trpc.identity.logout.useMutation()
+  const setLocation = useLocation()[1]
+
+  const login = () => {
+    setLocation("/auth/signin")
+  }
+
+  const logout = () => {
+    logoutMutation.mutate()
+  }
+
+  if (logoutMutation.isSuccess) checkLoginStatus.refetch()
+
+  return (
+    <div className="mt-5 pt-5 w-24 flex justify-center items-center">
```

Comment:
Why not use hier:
https://cloudoperators.github.io/juno/?path=/docs/layout-grid-grid--docs

Reply:
This is just a placeholder for the future layout.